app/src/main/java/org/oppia/android/app/resumelesson/ResumeLessonFragmentPresenter.kt

```diff
@@ -126,6 +126,10 @@ class ResumeLessonFragmentPresenter @Inject constructor(
       resumeLessonViewModel.chapterSummary.get()!!.summary,
       binding.resumeLessonChapterDescriptionTextView
     )
+    if (binding.resumeLessonChapterDescriptionTextView.text.toString().isBlank()) {
+      binding.resumeLessonChapterDescriptionTextView.importantForAccessibility =
+        View.IMPORTANT_FOR_ACCESSIBILITY_NO
```

Comment:
@vrajdesai78 As per discussion we should add two tests (1.) for `View.IMPORTANT_FOR_ACCESSIBILITY_YES` and  (2.) for View.IMPORTANT_FOR_ACCESSIBILITY_NO